pages/common/nrm.md

```diff
@@ -0,0 +1,27 @@
+# nrm
+
+> Nrm(npm registry manager) can help you easy and fast switch between different npm registries, now include: npm, cnpm, taobao, nj(nodejitsu), rednpm.
```

Comment:
Make the first line - NPM registry manager.
Then the second line can be - `Helps to easily switch between different npm registries.`

Since we can already list the registries, there is no need to mention it in the heading.